src/Control/Lens/Internal/Setter.hs

```diff
@@ -1,8 +1,6 @@
 {-# LANGUAGE CPP #-}
 
-#if __GLASGOW_HASKELL__ < 708
 {-# LANGUAGE Trustworthy #-}
```

Comment:
Previously, this line was only reachable on pre-7.8 versions of GHC, but now it's used unconditionally. Did you intend to make this change?

Reply:
Yes. I think this was something on the lines "inferred safe / trustworthy". Unconditional `Trustworthy` is clearer.